sycl-fusion/jit-compiler/lib/fusion/FusionHelper.cpp

```diff
@@ -64,6 +64,7 @@ Expected<std::unique_ptr<Module>> helper::FusionHelper::addFusedKernel(
     // correct name and signature only during the fusion pass.
     auto *F = Function::Create(FT, GlobalValue::LinkageTypes::ExternalLinkage,
                                "fused_kernel", *NewMod);
+    F->IsNewDbgInfoFormat = UseNewDbgInfoFormat;
```

Comment:
Seeing a kernel fusion commit in this git branch which i don't think should be here

Reply:
This should be gone now